marklogic-data-hub-central/ui/src/config/tooltips.config.tsx

```diff
@@ -484,7 +484,9 @@ const RunToolTips = {
   runEmptyFlow: 'There are no steps in the flow. You must add steps to the flow.',
   flowEdit: 'Edit Flow',
   flowDetails: 'Flow Details',
-  moveLeft: 'Move left'
+  moveLeft: 'Move left',
```

Comment:
Minor thing here but the tooltips here look to be the outdated ones. Please refer to the right column of that section in Matthew's UI Text wiki here: https://wiki.marklogic.com/display/ENGINEERING/Data+Hub+5.7.1+UI+Content#DataHub5.7.1UIContent-RunFlows